### DISCUSSION TITLE
AP Band switching "broken"

### DISCUSSION THREAD

**Discussion by UltraHQ** [Mygod/VPNHotspot#305] [Q&A]
I'm not able to switch between 2,4GHz and 5GHz in the Android Hotspot settings anymore, it is grayed out/blocked.

I'm speculating, this may have a connection with the app. Is this possible? Could this theoretically happen somehow?

I'm on the Galaxy A72, OneUI 3.1/Android 11

**Comment by Mygod** [comment]
Maybe your system update disabled that feature? 🤔

Can you change it using this app?

**Reply by UltraHQ** [reply]
Yes, I can change it with the App. It will show the changed band in the android settings, but it is still blocked
It's unlikely a system update removed it. I mean, the button is still there showing the current band, but it is blocked / grayed out

**Comment by UltraHQ** [comment]
I just found out, one of my magisk modules created this glitch. If you have the same problem, disable all CSC related modules.
